sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1463,7 +1463,14 @@ def init(self):
         # Initialize xcvr table helper
         xcvr_table_helper = XcvrTableHelper()
 
-        self.load_media_settings()
+        fastboot_tbl = swsscommon.Table(xcvr_table_helper.state_db_host, 'FAST_REBOOT')
+        keys = fastboot_tbl.getKeys()
+
+        if "system" in keys:
+            self.log_info("Skip loading media_settings.json in case of fast-reboot")
```

Comment:
You should also check if the value is true to be safe. you can use following syntax:

if fastboot.tbl.get('system', False):
    # skip

Reply:
please disregard my example. But please check the value.